Flow.Launcher.Core/Plugin/JsonRPCPlugin.cs

```diff
@@ -27,23 +27,10 @@ internal abstract class JsonRPCPlugin : IPlugin, IContextMenu
         /// </summary>
         public abstract string SupportedLanguage { get; set; }
 
-        protected abstract string ExecuteQuery(Query query);
+        protected abstract Task<Stream> ExecuteQueryAsync(Query query, CancellationToken token);
```

Comment:
Is this change backwards compatible?

Reply:
The JSONRPCPlugin is in Flow, and what it is doing is that overriding the Query (or QueryAsync) to call the executable (or python) and communicate via console stream. So change here won't affect the compatibility of plugins.